platform/pulpcore/app/tasks/importer.py

```diff
@@ -1,14 +1,37 @@
 from gettext import gettext as _
-import os
 
 from celery import shared_task
 
 from pulpcore.app import models
+from pulpcore.app.apps import pulp_plugin_config
 from pulpcore.tasking.services import storage
 from pulpcore.tasking.tasks import UserFacingTask
 
 
 @shared_task(base=UserFacingTask)
+def update(importer_id, app_label, serializer_name, data=None, partial=False):
+    """
+    Update an :class:`~pulp.app.models.Importer`
+
+    Args:
+        importer_id (basestring): the id of the importer
+        app_label (str): the Django app label of the plugin that provides the importer
```

Comment:
str for Python 3 https://docs.python.org/3.0/whatsnew/3.0.html#text-vs-data-instead-of-unicode-vs-8-bit